src/components/TextInput/TextInput.styles.css.ts

```diff
@@ -0,0 +1,61 @@
+import { style } from '@vanilla-extract/css';
+
+const COLOR_ACTIVE = '#2C98FC';
+
+const textInputContainer = style({
+  position: 'relative',
+  border: '1px solid grey',
```

Comment:
Use some color from `theme.palette.neutral`, or add a new one if there is no proper color in theme.
Same for `COLOR_ACTIVE`